play.py

```diff
@@ -50,9 +63,9 @@ def do_GET(self):
         output = [
             int(joystick[0] * 80),
             int(joystick[1] * 80),
-            int(round(joystick[2])),
-            int(round(joystick[3])),
-            int(round(joystick[4])),
+            int(joystick[2]),
+            int(joystick[3]),
+            int(joystick[4])
```

Comment:
would it be possible to implement this as a single function call that we use here instead of round? I'd like to explore that rather than adding all this code into the main loop.